docs/blog/2018-07-07-the-gatsby-plugin-page-creator/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Automatically create pages from components in any directory
+date: "2018-07-07"
+image: "gatsbymanor.jpg" # change this to the npm package stats
+author: "Steven Natera"
+tags: ["themes", "starters", "gatsby-manor"]
+---
+
+![Gatsby plugin page creator in the plugin library](plugin-library-page-creator.png)
+
+In this article we will talk about a new plugin that let’s you auto create
+pages from components in any folder within your Gatsby project! We will chat about the
+problem that sparked the creation of this plugin. Afterwards, we'll talk about how you can
+use this plugin to auto create pages without having to place your components in the **src/pages** directory.
+
+# The problem with static values
+Here's how it all got started. The goal with [Gatsby Manor](https://gatsbymanor.com)
```

Comment:
Could you make it clear this is a separate project you run? Sometimes people think Gatsby Manor is an official project so we should make it clear it's not.

Reply:
For sure! Thanks for letting me grow along side the community. I think the more starters we make the more people can jump onto the Gatsby wagon!